openff/qcsubmit/exceptions.py

```diff
@@ -217,3 +217,21 @@ class RecordTypeError(QCSubmitException):
 
     error_type = "record_type_error"
     header = "Record Type Error"
+
+
+class InvalidConvergeSettingsError(AssertionError):
+    """
+    Invalid converge options were passed to GeometricProcedure()
+    """
+
+    error_type = "invalid_converge_settings_error"
+    header = "Invalid Converge Settings Error"
+
+
+class ConflictingConvergeSettingsError(ValueError):
```

Comment:
```suggestion
class ConflictingConvergeSettingsError(QCSubmitException):
```

Just going by what else in in this class, I think you meant to inherit this way

Reply:
According to the docs, a Pydantic validator can only raise a `ValueError` or `AssertionError`, not any other type of error. @JoshHorton suggested that I should make custom exceptions for these cases, rather than just use the basic `ValueError` or `AssertionError`, but I _think_ that means that they can't inherit from the `QCSubmitException`. If we prefer all custom exceptions to inherit from `QCSubmitException`, I can just revert these to generic `ValueError` or `AssertionError`.

Reply:
Ah yeah I see that now. I'd suggest either
* inheriting from both (not sure the proper order) if it works
* simply dropping the `error_type` and `header` lines, since they probably won't get used if not inheriting from the custom type and I think they're only used for pretty-ifying the error message